clients/src/main/java/org/apache/kafka/clients/consumer/Consumer.java

```diff
@@ -249,7 +249,7 @@
     ConsumerGroupMetadata groupMetadata();
 
     /**
-     * @see KafkaConsumer#enforceRebalance)
+     * @see KafkaConsumer#enforceRebalance()
```

Comment:
This has been fixed by: https://github.com/apache/kafka/pull/8291 
Please omit and rebase to get the latest changes.